doc/nrf/device_guides/working_with_nrf/nrf70/developing/debugging.rst

```diff
@@ -37,3 +37,33 @@ With CMake
 
     cmake -GNinja -Bbuild -DBOARD=nrf7002dk_nrf5340_cpuapp -DSNIPPET=nrf70-debug samples/wifi/shell
     ninja -C build
+
+Statistics
+==========
+
+The nRF70 driver, firmware, and networking stack have statistics that can be enabled to help debug issues.
+
+You can enable statistics by either using ``nrf70-debug`` snippet, see `Enable debug features`_.
+
+.. list-table:: Statistics table
+    :header-rows: 1
+
+    * - Command
+      - Description
+      - Functional area
+    * - ``net stats``
+      - Displays statistics for the networking stack, network interfaces, and network protocols.
+      - Data path debugging (Networking stack)
+    * - ``wifi statistics``
```

Comment:
Would like to confirm if it is ``wifi stats`` or ``wifi statistics``?
```suggestion
    * - ``wifi statistics``
```

Reply:
what is the diff here, both look same to me?

Reply:
The intent was not to delete anything but just added a question. Please ignore this.